src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
Could we add these as class options in stead of the `inverseStyle` options, since this will never change after definition:
```js
FilterByTaskState.defaultProps = {
  buttonClassName: 'button dropdown-toggle text-align-left',
  dropdownMenuClassName: 'dropdown-menu'
  ...
}
```

That way you have full control over it:
```js
<FilterByTaskState
    buttonClassName="button button-inverse dropdown-toggle text-align-left"
    dropdownMenuClassName="dropdown-menu inverse"
    ... />
```